Jenkinsfile

```diff
@@ -375,7 +382,7 @@ def buildAndroid() {
   }
 }
 
-def publish(nodeVersions, dependencies, tag) {
+def publish(nodeVersions, electronVersions, dependencies, tag) {
```

Comment:
Is electronVersions used at all?

Reply:
Yes, there is a loop in the function where it unstash the Electron builds.